sdk/identity/Azure.Identity/tests/IdentityTestEnvironment.cs

```diff
@@ -31,7 +31,7 @@ public class IdentityTestEnvironment : TestEnvironment
 
         public string TestPassword => GetOptionalVariable("AZURE_IDENTITY_TEST_PASSWORD") ?? "SANITIZED";
         public string TestTenantId => GetRecordedOptionalVariable("TENANT_ID") ?? GetRecordedVariable("AZURE_IDENTITY_TEST_TENANTID");
-        public string KeyvaultScope => GetRecordedOptionalVariable("AZURE_KEYVAULT_SCOPE") ?? TestEnvVar.KeyVaultDefaultScope;
+        public string KeyvaultScope => GetRecordedOptionalVariable("AZURE_KEYVAULT_SCOPE") ?? "https://vault.azure.net/.default";
```

Comment:
Why the revert back to a hardcoded value?

Reply:
The hardcoded value of AZURE_KEYVAULT_SCOPE was previously defined in the TestEnvVar.cs file, but the file was deleted now. Should I add TestEnvVar.cs file and use the previous method?

Reply:
If removing `?? "https://vault.azure.net/.default"`, some tests will fail. I used `SANITIZED` instead of hardcode value, it can also pass.